crates/uv-workspace/src/workspace.rs

```diff
@@ -62,6 +62,8 @@ pub struct Workspace {
     ///
     /// This table is overridden by the project sources.
     sources: BTreeMap<PackageName, Source>,
+    /// The `pyproject.toml` of the workspace root.
+    pyproject_toml: PyProjectToml,
```

Comment:
@konstin - Any reason not to include this? It could be the same as the root member, but for virtual workspaces I can't see another way to get it.